internal/cli/main.go

```diff
@@ -125,6 +126,10 @@ func Main(args []string) int {
 	// Run the CLI
 	exitCode, err := cli.Run()
 	if err != nil {
+		if strings.Contains(err.Error(), "already installed") {
```

Comment:
Might be nice to add a comment here explaining what this is catching, and maybe even linking to https://github.com/posener/complete/blob/9a4745ac49b29530e07dc2581745a218b646b7a3/cmd/install/zsh.go#L21.

Reply:
Good idea, I added a comment in fcd37afe6 which links to this issue as opposed to the `zsh.go` file mentioned because that's dependent on the users shell (could be bash, etc).